stdlib/lexing.mli

```diff
@@ -108,6 +108,15 @@ val from_function : ?with_positions:bool -> (bytes -> int -> int) -> lexbuf
    starting at index 0, and return the number of bytes
    provided. A return value of 0 means end of input. *)
 
+val set_position : lexbuf -> position -> unit
```

Comment:
I'm slightly uneasy at accepting a record where a field is ignored, but I'm also not particularly familiar with doing it. Why is

```OCaml
val set_position : lexbuf -> int -> int -> int -> unit
```
not more appropriate? (with appropriate discussion on labelling, or using a triple)?

Reply:
I wanted this function be compatible with

`val set_position: lexbuf -> Lexing.position -> unit` in [Sedlexing](https://github.com/ocaml-community/sedlex/blob/master/src/lib/sedlexing.mli#L51)

Reply:
I think it's OK to ignore one field of the record. At any rate, better than passing a triple or three unlabeled arguments.